src/parsers/en/ENTimeLaterFormatParser.js

```diff
@@ -21,20 +27,25 @@ exports.Parser = function ENTimeLaterFormatParser(){
     };
 
     this.extract = function(text, ref, match, opt){
-
         if (match.index > 0 && text[match.index-1].match(/\w/)) return null;
 
+        var prefix = match[2];
+        var suffix = match[4];
+        if (!prefix && !suffix) return null;
```

Comment:
it didn't seem worth worrying about when both the prefix and suffix were there, it makes sense in some phrases, but not all:
-  (a little weird, but seems fine):
  - "in 3 minutes later"
  - "in 3 minutes henceforth
- (seems good) "in 3 minutes from now"

etc.